frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Card, CardHeader, CardBody } from '@patternfly/react-core';
+import { StarIcon } from '@patternfly/react-icons';
+import './SelectorCard.scss';
+
+interface SelectorCardProps {
+  title: string;
+  iconUrl: string;
+  name: string;
+  displayName: string;
+  selected: boolean;
+  recommended?: boolean;
+  onChange: (name: string) => void;
+}
+
+const SelectorCard: React.FC<SelectorCardProps> = ({
+  title,
+  iconUrl,
+  name,
+  displayName,
+  selected,
+  recommended = false,
+  onChange,
+}) => {
+  const classes = classNames('odc-selector-card', { 'is-selected': selected });
+  return (
+    <Card
+      component="button"
+      type="button"
+      aria-label={title}
+      className={classes}
+      onClick={() => onChange(name)}
+    >
+      <CardHeader>
+        <img className="odc-selector-card__icon" src={iconUrl} alt={displayName} />
+      </CardHeader>
+      <CardBody>
+        <span className="odc-selector-card__title">{title}</span>
```

Comment:
I know this is old code, but it's extremely odd that the display name is used as an image alt.
Perhaps rename to `fullName`.

Reply:
yes but i think it's referenced from annotations 

```
const displayName = _.get(imageTag, ['annotations', 'openshift.io/display-name'], defaultName);
```

https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/utils/imagestream-utils.ts#L128

i need to change across let me know as it may be confusing based on annotations name